packages/cerebral/src/Controller.js

```diff
@@ -67,7 +68,7 @@ class Controller extends EventEmitter {
   */
   flush () {
     const changes = this.model.flush()
-    const computedsAboutToBecomeDirty = computedDependencyStore.getUniqueEntities(changes)
+    const computedsAboutToBecomeDirty = this.strictRender ? computedDependencyStore.getStrictUniqueEntities(changes) : computedDependencyStore.getUniqueEntities(changes)
```

Comment:
standard allows multiline here, just question mark and colon should start a new line